elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -77,4 +79,7 @@
 			value = "Elasticsearch connection time-out; check the cluster status, it should be 'green';\n Request:\n========\n%1$sResponse:\n=========\n%2$s" )
 	SearchException elasticsearchRequestTimeout(String request, String response);
 
+	@Message(id = ES_BACKEND_MESSAGES_START_ID + 11,
+			value = "Projection of non-JSON-primitive field values is not supported: '%1$s'")
+	SearchException unsupportedProjectionOfNonJSONPrimitiveFields(JsonElement value);
```

Comment:
Minor nitpicking: we use the case "Json" everywhere as we conform to the Gson convention. I'm only talking about the method name, the message is fine.